python/BioSimSpace/Sandpit/Exscientia/_SireWrappers/_residue.py

```diff
@@ -33,7 +33,8 @@
 import random as _random
 import string as _string
 
-from Sire import Mol as _SireMol
+import sire.legacy as _Sire
```

Comment:
Is the outlook to eventually change all of these imports to their new equivalents, rather than `sire.legacy`?

Reply:
Yes, eventually we plan to move the new API, and likely modernise the API of BioSImSpace too. (We will also support the old API for some period too.) After this merge we will create a 2023.0.0 pre-release of BioSimSpace, with the intention of updating things in time for the full 2023.1.0 release. At lot of the Sire updates will make it a lot easier to directly work with Sire objects too, so we might expose those more readily. (Sire will also have an improved website with new tutorials and documentation.)

(All of the Sire API change is done with some clever Python renaming behind the scenes, which we could also do for BioSimSpace too.)

One thing I do like about the existing Sire API is that it mirrors the C++ code, so if you know one, then it's easy to translate things to the other. However, since most external development work will be done in Python land, it makes sense to (where possible) follow standards there.

Reply:
This sounds good, thanks @lohedges .